index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+Preferred method of contact<!DOCTYPE html>
```

Comment:
Sorry, my bad, yes I did it accidentally. I am removing and letting you know in couple of minutes.